apig/generate.go

```diff
@@ -110,6 +137,26 @@ func camelToLowerCamel(s string) string {
 	return strings.Join(ss, "")
 }
 
+func camelToOriginal(s string) string {
```

Comment:
It is a compound word of space-separated.

original: `account name`
lowerCamel: `accountName`
snake_case: `account_name`